chart/templates/default-backupstore-settings.yaml

```
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: longhorn-default-backupstore-settings
```

To prevent any confusion, remove `settings`
```suggestion
  name: longhorn-default-backupstore
```

---

Modified. Thanks.